lib/archethic/mining.ex

```diff
@@ -126,7 +126,7 @@ defmodule Archethic.Mining do
         Archethic.TaskSupervisor,
         storage_nodes,
         &P2P.send_message(&1, message),
-        max_concurrency: nb_storage_nodes,
+        max_concurrency: max(System.schedulers_online(), nb_storage_nodes),
```

Comment:
nb_storage_nodes cannot be 0 so it is better to set the max_concurrency to nb_storage_nodes than scheduler online